### PR TITLE
Kriptonian osint

### LINKED ISSUE #45: OSINT Tool
## Suggest a new script for the project
OSINT Tool to find phone number and IP address info

### Describe the new script you want
Use python modules to find information about IP address and phone number to help in OSINT investigation 

## Do you want to work on this?
- [x] yes
- [ ] no

### PR DESCRIPTION
**Pull request by kriptonian1**
# Description

OSint tool addded

Fixes #45 

## Type of change

- [x] New Script (non-breaking change which adds a new script to the repository)

## Has This Been Tested?

- [ ] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [ ] I have gone through the contributing guidelines before making this Pull Request.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: scripts/Password_Manager/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+black==19.10b0
+cryptography @ file:///tmp/build/80754af9/cryptography_1616769286105/work
```
Comment (rijusougata13) [inline scripts/Password_Manager/requirements.txt:2]: Fix this!

**Review by rijusougata13 — CHANGES_REQUESTED** [review]

File: scripts/Password_Manager/requirements.txt (hunk 1 above)
Comment (kriptonian1) [reply scripts/Password_Manager/requirements.txt:2]: Okey

Comment (OBITORASU) [reply scripts/Password_Manager/requirements.txt:2]: Remove the password manager files from this PR

File: scripts/Password_Manager/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<h1 align="center"> Simple Password Manager </h1>
+
+Simple Password manager to store password securely 
+## Setup 
+- Run `pip3 install -r requirements.txt` to install dependency 
+- To run the file `python3 main.py`
```
Comment (OBITORASU) [inline scripts/Password_Manager/README.md:6]: This file should be removed

File: scripts/Password_Manager/main.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    elif command == "3":
+        data = input("Enter site name: ")
+        user, passwd = Print_Passwd(data)
+        print(user, "|", passwd)
+
+    elif command == "exit":
+        break
+
+    else:
+        print("No such command exist")
```
Comment (OBITORASU) [inline scripts/Password_Manager/main.py:106]: This file should also be removed.

**Comment by OBITORASU** [comment]
Closing this PR because of the messed up commits, please make a fresh one :)

**Pull request closed without merging**